### PR TITLE
[zh] Add zh version of homepage

### PR DESCRIPTION
**Pull request by windsonsea**
- Contributes to #2402
- More context #4302 

Some tips:

- add `/zh` to its relevant url path if the Chinese version of the page is available.
- add anchors to headings in those pages under `zh/docs/`
- `npx prettier --w` is not so perfect to Chinese. It's not correct to add any space between Chinese words. It's recommended to add a space after en words, numbers, and punctuations
- A localization style guide may be required

---

**Edit** (@chalin)

**Preview**: https://deploy-preview-4341--opentelemetry.netlify.app/zh/

**Redirect** tests:

- https://deploy-preview-4341--opentelemetry.netlify.app/en/
- https://deploy-preview-4341--opentelemetry.netlify.app/en/docs
- https://deploy-preview-4341--opentelemetry.netlify.app/schemas/latest

### CODE REVIEW

**Comment by svrnm** [comment]
Thank you @windsonsea! We will take your PR to work towards making the Chinese version available, so we will push commits to your PR, so if you want to apply any additional changes make sure you pull commits if there are any

>     * `npx prettier --w` is not so perfect to Chinese. It's not correct to add any space between Chinese words. It's recommended to add a space after en words, numbers, and punctuations

This is good to know, thanks. As a starting point we can disable prettier for the `zh` folder, but I would prefer to have it in place as soon as the translation grows, but we need your help here, can you check if there is a prettier configuration that would help with that? We currently have turned on `proseWrap: always`, which based on a quick search on https://github.com/prettier/prettier/issues might be part of the issue?
 
>     * A localization style guide may be required

Yes! The moment we have the first language landed, we can work on that.

**Comment by windsonsea** [comment]
Seems this issue is talking about Chinese prettier: https://github.com/prettier/prettier/issues/14936

We need to add some rules:

- Do not add newline between Chinese words
- Do not add newline before a Chinese punctuation symbol like `，。！？、` which are two-byte chars different from en symbols `, . ! ?`

**Comment by svrnm** [comment]
Thanks for taking a look and pointing to the right issue, I'll take a deeper look. Note, that prettier is unfortunately not very configurable, [see here](https://prettier.io/docs/en/option-philosophy), so I am not sure if we can implement that somehow, @chalin probably knows best.

**Comment by chalin** [comment]
FYI, I'll get working on this as soon as I can.

**Comment by chalin** [comment]
@windsonsea - could you please only include the translation of the homepage? We'll get to the other page later, but at the moment, it is just causing more trouble than is worth (because there isn't an index page for that section yet).

**Comment by windsonsea** [comment]
Hi @chalin , sure, only the _index.md is included now.

**Comment by chalin** [comment]
@windsonsea - is "OpenTelemetry 项目网站" a good translation for "The OpenTelemetry Project Site", and if not, what would be?

**Comment by windsonsea** [comment]
> @windsonsea - is "OpenTelemetry 项目网站" a good translation of "The OpenTelemetry Project Site", and if not, what would be?

Yes, it's fine.

File: content/zh/_index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+</div>
+{{< /blocks/cover >}}
+
+{{% blocks/lead color="white" %}}
+
+OpenTelemetry 是各类 API、SDK 和工具形成的集合。可用于仪表化、生成、采集和导出遥
+测数据（链路、指标和日志），帮助你分析软件的性能和行为。
+
+> OpenTelemetry 在[多种编程语言](/docs/languages/)均达到 **GA** 级别，普适性很高
+> 。
```
Comment (chalin) [inline content/zh/_index.md:48]: @windsonsea - can this be put all on the same line?

```suggestion
> OpenTelemetry 在[多种编程语言](/docs/languages/)均达到 **GA** 级别，普适性很高。
```

File: content/zh/_index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+{{% /blocks/lead %}}
+
+{{% blocks/section color="dark" type="row" %}}
+
+{{% blocks/feature icon="fas fa-chart-line" title="链路、指标、日志" %}}
+
+从你的服务和软件中创建并采集遥测数据，然后将其转发给各种分析工具
+。{{% /blocks/feature %}}
```
Comment (chalin) [inline content/zh/_index.md:57]:
```suggestion
从你的服务和软件中创建并采集遥测数据，然后将其转发给各种分析工具。

{{% /blocks/feature %}}
```

**Review by chalin — COMMENTED** [review]
@windsonsea PTAL at my inline comments

**Comment by chalin** [comment]
@windsonsea - I'm going to merge the suggested edits now because I want to merge this PR. We can always fix it in a followup PR.
@svrnm et al. - merging this PR now. Note that the zh pages won't show up in production b/c the pages are marked as draft atm.

**Review by chalin — APPROVED** [review]

**Comment by chalin** [comment]
In case you missed it in the opening-comment edit I made, you can see a preview of the Chinese homepage:

- https://deploy-preview-4341--opentelemetry.netlify.app/zh/

**Comment by windsonsea** [comment]
> In case you missed it in the opening-comment edit I made, you can see a preview of the Chinese homepage:
> 
> * https://deploy-preview-4341--opentelemetry.netlify.app/zh/

Cool! The preview looks good. 👍

**Pull request merged**